src/themes/themes.css

```diff
@@ -0,0 +1,115 @@
+/* A basic color theme and associated dark theme inspired by
+   https://material.io/design/color/dark-theme.html.
+
+ Can be imported via:
+
+    @import '~elements-sk/themes/themes';
+*/
+
+@import url(./color-palette.css);
+
+/* Define Color Palette */
+html {
+  /* See https://material.io/design/color/dark-theme.html for how to use each
+     of the below. */
+  --primary: var(--blue-800);
```

Comment:
It would be great to have comments explaining the criteria and tools used to choose the colors below. Preferably in an actionable way, so that we know the reasoning to follow if we need to make changes in the future.

For example, if the [Material Design Color Tool](https://material.io/resources/color) was used, please indicate the primary and secondary colors entered into the tool, and how the colors generated by the tool are used below (i.e. the light/dark variations of the primary/secondary colors).

If these colors were chosen by hand, please briefly mention the criteria used.

Same for the dark theme.

Reply:
Done.